### PR TITLE
Add section about Producer configs

### PR DESCRIPTION
**Pull request by morenol**
image

### CODE REVIEW

**Comment by morenol** [comment]
> Are these fields configurable in the connector? If yes, we should also update in the connector section.

not yet

**Comment by sehz** [comment]
Can you add a section on how various config settings impacts performance?

File: content/docs/clients/producer.md [hunk 1]
```diff
@@ -80,18 +80,16 @@ on the properties we just covered:
...
+- `batch_size`: Indicates the maximum amount of bytes that can be accumulated in a batch.
+- `linger`: Time to wait before sending messages to the server. Defaults to 100 ms.
+- `compression`: Compression algorithm used by the producer to compress each batch before sending it to the SPU. Supported compression algorithms are none, gzip, snappy and lz4.
+  
+In general, each one of these configurations has its drawbacks. For instance, with the compression algorithm, it is a trade-off between disk usage in the server and CPU usage in the producer and the consumer for compression and decompression. Typically, the compression ratio is improved when the payload is large, therefore a larger `batch_size` could be used to improve the compression ratio. A `linger` equals `0` means that each record is sent as soon as possible. A `linger` time larger than zero introduces latency but improves throughput.
```
Comment (sehz) [inline content/docs/clients/producer.md:90]: change `drawback` to something more positive.

**Review by sehz — CHANGES_REQUESTED** [review]
a minor nit

File: content/docs/clients/producer.md (hunk 1 above)
Comment (ajhunyady) [reply content/docs/clients/producer.md:90]: How about this: "In general, each one of these configurations has a benefit and a potential drawback." @sehz @morenol

**Review by sehz — APPROVED** [review]
Good enough for now

**Pull request merged**